src/Tools/dotnet-monitor/Strings.resx

```diff
@@ -352,7 +352,7 @@
     <comment>Gets the string to display in help that explains what the '--urls' option does.</comment>
   </data>
   <data name="HelpDescription_OutputFormat" xml:space="preserve">
-    <value>The format to display the output in. Valid values are Json, Text, and EnvVar.</value>
+    <value>The format to display the output in. This option is useful for printing the information in a format to be copy/pasted into the system of your choice.</value>
```

Comment:
Feels a bit over explaining. How about replacing the entire description with something like `The output format of the API Key configuration.`

Reply:
This is too specific. The intention is that this parameter can be a common parameter attached to any command that can write out information in multiple formats. 
Not that I expect this to ever be useful, but I intended this to be re-usable if we made `dotnet monitor config show --output PowerShell` write out the configuration block as environment variables.

Reply:
Keep the help text as specific as it is used right now and don't over generalize for something we may or may not do in the future.

Reply:
I agree with Justin in that this seems over explained.
My 🌶️ take is that I prefer the original

Reply:
We can always change it again later for the new usages of the text.

Reply:
Updated to take @jander-msft 's suggestion.